### PR TITLE
Add ability to download multiple files from a customized url

### PR DESCRIPTION
**Pull request by jdekonin**
Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

File: get.sh [hunk 1]
```diff
@@ -149,12 +149,15 @@ getBinaryOpenjdk()
...
-		if [ $? -ne 0 ]; then
-			echo "Failed to retrieve the jdk binary, exiting"
-			exit 1
-		fi
+		for file in $download_url
+		do 
+			echo "curl -OLJks ${curl_options} $file"
+			curl -OLJks ${curl_options} $file
+			if [ $? -ne 0 ]; then
+				echo "Failed to retrieve the jdk binary, exiting"
```
Comment (smlambert) [inline get.sh]: Should update both line 44 (with a note about format of customizedURL, if multiple files present or a note about the fact its supported) and line 157, to be more useful, failed to retrieve $file, or something like that.

**Comment by jdekonin** [comment]
@llxia ready for review

File: get.sh (hunk 1 above)
Comment (jdekonin) [reply get.sh]: Updates made.

**Comment by llxia** [comment]
@jdekonin Please remove my changes as it is WIP and remove the change in systemtest/playlist.xml.

**Comment by jdekonin** [comment]
Sorry hadn't meant to leave that in there.  I was only trying to see if testing would go further with both your changes and mine.  It has been cleaned up.

**Comment by llxia** [comment]
No worries. Thanks for testing my changes. :)

**Comment by sophia-guo** [comment]
Collaborated #542

**Pull request merged**